ngraph/frontend/onnx_import/src/op/sub.hpp

```diff
@@ -26,22 +26,33 @@ namespace ngraph
                     {
                         if (node.has_attribute("axis"))
                         {
+                            NGRAPH_CHECK((lhs_node.get_partial_shape().rank().is_static() &&
+                                          rhs_node.get_partial_shape().rank().is_static()),
+                                         "Sub input's rank has to be static.");
                             // Unidirectional broadcast right node to left shape.
-                            const auto axis = node.get_attribute_value<std::int64_t>("axis");
-                            const auto axes_mapping = builder::opset1::get_axes_mapping_output(
-                                lhs_node.get_partial_shape(), rhs_node.get_partial_shape(), axis);
-                            rhs_node = std::make_shared<default_opset::Broadcast>(
-                                rhs_node,
-                                std::make_shared<default_opset::ShapeOf>(lhs_node),
-                                axes_mapping);
+                            auto axis = node.get_attribute_value<std::int64_t>("axis");
```

Comment:
Am I right that we have code duplication for each file?
If yes, can we create a common function?